### DISCUSSION TITLE
Guys help out with a link or info

### DISCUSSION THREAD

**Discussion by lookWhatIvanDid** [Chia-Network/chia-blockchain#6851] [Q&A]
I have NO SYNC problem always. I've never seen it done yet. I mean it stops when I start making a plot, and never gets fully sync when I don't.

Its rather common problem that has no answer yet, or its too much questions like mine and I still cant find the answer.
I did open 8444 port tho, I think so...

**Comment by lvcivs** [comment]
Plotting has nothing to do with it. 

I recommend to read the FAQ about connections and follow the tips mentioned there: https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#why-does-my-node-have-no-connections-how-can-i-get-more-connections
